deps/uv/src/unix/fs.c

```diff
@@ -383,7 +392,7 @@ static ssize_t uv__fs_pathmax_size(const char* path) {
 #if defined(PATH_MAX)
     return PATH_MAX;
 #else
-    return 4096;
+#error "PATH_MAX undefined in the current platform"
```

Comment:
There may be no frosting *at all*, depending on whether you called `process.exit()` too soon.